libs/features/mas/commerce/src/checkout-link.js

```diff
@@ -121,6 +121,7 @@ export class CheckoutLink extends HTMLAnchorElement {
                 if (countryCode) this.dataset.imsCountry = countryCode;
             }, ignore);
         }
+        overrides.imsCountry = null;
```

Comment:
do we need this line?

Reply:
@yesil Yes, this will annul the IMS country in options and then the "page country" will be used, the same country that was used in the first render. And since we already have offers loaded for that country, no new WCS call will be triggered.